### DISCUSSION TITLE
`--replace` flag not being passed when using `yarn documents create`

### DISCUSSION THREAD

**Discussion by coreyward** [sanity-io/sanity#3442] [Sanity Studio Feedback]
I'm running a command like `yarn sanity documents create --id <document-id> --replace` to manually remove part of a document that is causing the V3 studio to crash, but after saving and closing the file the command returns an error:

> Failed to write documents: the mutation(s) failed: Document by ID "<document-id>" already exists
> Perhaps you want to use `--replace` or `--missing`?

I also tried moving the `--replace` flag to before the `--id` flag, but the behavior is the same. And finally, I tried running with `./node_modules/sanity/bin/sanity` directly in case `yarn` was swallowing the arg, but again, same behavior.

**Comment by rexxars** [comment]
Fixed in [v3.0.0-dev-preview.12](https://github.com/sanity-io/sanity/releases/tag/v3.0.0-dev-preview.12), thanks for reporting!
